### DISCUSSION TITLE
将操作的“编辑”和“删除”功能分一开点

### DISCUSSION THREAD

**Discussion by Jeamorg** [Qexo/Qexo#302] [Ideas]
“编辑”和“删除”功能靠得太近容易误点，能否考虑一下将它们分开一点？

![](https://img.jeam.cc/other/Snipaste_2023-01-21_10-55-43.png)

**Comment by am-abudu** [comment]
7ce11c86c9fd3d7840f32f78e9e20822eb28acde

**Comment by ning0818** [comment]
不会误点的
![image](https://user-images.githubusercontent.com/117955401/227701595-636c1201-1726-42f2-909c-f03517bc075a.png)
